Libplanet/Net/Protocols/KademliaProtocol.cs

```diff
@@ -57,6 +60,8 @@ public class KademliaProtocol : IProtocol
                 requestTimeout ??
                 TimeSpan.FromMilliseconds(5000);
             _transport.ProcessMessageHandler += ProcessMessageHandler;
+            _cache = new ConcurrentDictionary<Address, BoundPeer>();
+            _cacheSize = int.MaxValue;
```

Comment:
Is it okay we always use `int.MaxValue`?

Reply:
I have no idea for this value actually... Do you have suggestion?

Reply:
Can we use the value related `Kademlia.TableSize` or `Kademlia.BucketSize` here?